client/main.go

```diff
@@ -35,6 +47,7 @@ var (
 )
 
 func main() {
+	flag.Var(&runtimeVars, "envs", "Runtime environment variables")
```

Comment:
Should we define this above like the rest of the flags?

Reply:
Unfortunately Var function does not have a return type, to cannot define it like the rest of the flags-  https://pkg.go.dev/flag#Var